src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/fields/ResourceLocator.js

```diff
@@ -135,6 +151,9 @@ class ResourceLocator extends React.Component<FieldTypeProps<?string>> {
                 </div>
                 {formInspector.id &&
                     <div className={resourceLocatorStyles.resourceLocatorHistory}>
```

Comment:
This causes this stuff to be styled. There should be a `resourceLocator.scss` file in the same folder, which contains a class named like this property here. Should probably be renamed, because it is not only the `resourceLocatorHistory` anymore.